components/src/core/info-list-item/info-list-item.component.ts

```diff
@@ -79,17 +85,28 @@ type DividerType = 'full' | 'partial' | undefined;
     },
 })
 export class InfoListItemComponent implements AfterViewInit {
-    @Input() statusColor: string;
+    /** Show a colored background for the icon */
+    @Input() avatar = false;
+    /** Add a chevron icon on the right */
     @Input() chevron = false;
+    /** Smaller height row with less padding */
     @Input() dense = false;
-    @Input() avatar = false;
+    /** Show a row separator below the row */
+    @Input() divider: DividerType;
+    /** Disable the list item */
+    @Input() disabled = false;
+    /** Remove left padding if no icon is used */
     @Input() hidePadding = false;
+    /** Icon alignment when avatar is set to false */
     @Input() iconAlign: IconAlignType = 'left';
+    /** Left border color */
+    @Input() statusColor: string;
+    /** Whether to wrap info on overflow */
```

Comment:
These should specifically say something about setting the text wrapping rather than adding a wrapper around the text (there was some confusion from adopter teams about this)

Reply:
I've completely changed the description in the javadocs and also the READMEs; they are slightly different because of the length restrictions in the README but it's more specific now.